src/schema/nmdc.yaml

```diff
@@ -143,13 +143,36 @@ subsets:
       - Assignment of the data_portal_subset is incomplete in the schema.
 
 classes:
-
+  MetaPFunctionAggMember:
+    description:
+      A class for the metaproteomics aggregation table, used to enable NMDC's data portal KEGG search
```

Comment:
ideally, `description`s say what an element is (in reality), they don't reiterate an element's type (class) or how it will be used. (That second part is better suited for a comment)

An example of this would be 'A count of the number of times a sample (or gene or whatever) has been annotated with the specified gene function"